protoactor/remote.py

```diff
@@ -0,0 +1,157 @@
+from protoactor.actor import Actor, from_producer
+from protoactor.pid import PID
+from protoactor.context import AbstractContext
+from protoactor.messages import Started, Terminated
+from protoactor.process_registry import ProcessRegistry
+
+
+class Endpoint(object):

```

Comment:
No worries. That's why we have code reviews, to help each other write better code and maintain the same level of code quality.